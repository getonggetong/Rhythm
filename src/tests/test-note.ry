init() {
	
	print(G4);
	print(G#4);
	print(Gb4);
	print(Gb4.4);

}