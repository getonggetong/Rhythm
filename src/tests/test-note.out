G4
G#4
Gb4
Gb4.4